react/features/base/conference/middleware.native.ts

```diff
@@ -1 +1,39 @@
+
+import { readyToClose } from '../../mobile/external-api/actions';
+import { showWarningNotification } from '../../notifications/actions';
+import { NOTIFICATION_TIMEOUT_TYPE } from '../../notifications/constants';
+import { JitsiConferenceErrors } from '../lib-jitsi-meet';
+import MiddlewareRegistry from '../redux/MiddlewareRegistry';
+
+import { CONFERENCE_FAILED } from './actionTypes';
+import { leaveConference } from './actions';
+import {
+    CONFERENCE_DESTROYED_LEAVE_TIMEOUT,
+    TRIGGER_READY_TO_CLOSE_REASONS
+} from './constants';
+
 import './middleware.any';
+
+MiddlewareRegistry.register(store => next => action => {
```

Comment:
Native should have the same behavior as web: a dialog that shows the reason and an OK button to press. On button press we'd go to the welcome page, since there is no feedback dialog on mobile yet.